pipelines/SpeciesTreeFromBusco/scripts/fetch_genomes_from_db.py

```diff
@@ -0,0 +1,109 @@
+#!/usr/bin/env python3
+
+# See the NOTICE file distributed with this work for additional information
+# regarding copyright ownership.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+Locate genome dumps of a collection or a species set on disk.
+Example:
+    $ python fetch_genomes_from_db.py -u mysql://ensro@mysql-ens-compara-prod-1:4485/ensembl_compara_master -c pig_breeds -o test.tsv -d /hps/nobackup/flicek/ensembl/compara/shared/genome_dumps/vertebrates
+"""
+
+import sys
+import argparse
+from os import path
+from sqlalchemy.engine.row import Row
+from sqlalchemy import create_engine, text
+
+# Parse command line arguments:
+parser = argparse.ArgumentParser(
+    description='Locate genomes of a species set/collection in the dump directory.')
+parser.add_argument('-u', metavar='db_URL', type=str, help="Compara master db URL.", required=True)
+parser.add_argument(
+    '-d', metavar='dump_dir', type=str, help="Genome dump directory.", required=True)
+parser.add_argument(
+    '-s', metavar='ssid', type=str, help="Species set id.", required=False, default=None)
+parser.add_argument(
+    '-c', metavar='cid', type=str, help="Collection id.", required=False, default=None)
+parser.add_argument(
+    '-o', metavar='output', type=str, help="Output tsv.", required=True)
```

Comment:
For consistency with `fix_leaf_names.py`. 🙂
```suggestion
    '-o', metavar='output', type=str, help="Output CSV.", required=True)
```